src/gui/dialogs/preferences_dialog.cpp

```diff
@@ -411,11 +413,11 @@ void preferences_dialog::post_build(window& window)
 		font_scaling, set_font_scaling);
 
 	/* SELECT THEME */
-	connect_signal_mouse_left_click(
-			find_widget<button>(&window, "choose_theme", false),
-			bind_void(&show_theme_dialog,
-			std::ref(window.video())));
-
+	button *theme_button = find_widget<button>(&window, "choose_theme", false, false);
+	if (theme_button != nullptr) {
+		connect_signal_mouse_left_click(
+				*theme_button, bind_void(&show_theme_dialog, std::ref(window.video())));
+	}
```

Comment:
I think it may not be necessary to disable theme switching (which just dictates the layout of the in-game GUI), though I kind of understand why you would. Note that the theme can be switched by scenario WML too though; because of this, it might actually be advisable to remove/replace some of the existing themes when running on iOS.